libssh2-sys/lib.rs

```diff
@@ -494,6 +494,9 @@ extern {
     pub fn libssh2_scp_recv(sess: *mut LIBSSH2_SESSION,
                             path: *const c_char,
                             sb: *mut libc::stat) -> *mut LIBSSH2_CHANNEL;
```

Comment:
interestingly, `libc::stat` appears to be equivalent to `_stati64` for the msvc targets, which is in turn what libssh2 chooses in the default windows flavor of the build for its `libssh2_struct_stat` type.

Since that is a distinct type from `libc::stat`, and there doesn't appear to be a `libc::_stat` that we can reference, I feel like it would be a good idea to break `libssh2_scp_recv` on windows so that we can't call it now that we know that the type is wrong.

In other words, stick `#[cfg(not(windows))]` as an attribute on this to effectively nerf it.